discord-service/src/main/java/pl/flyingoctopus/discord/member/action/BriefAction.java

```diff
@@ -0,0 +1,54 @@
+package pl.flyingoctopus.discord.member.action;
+
+import discord4j.core.object.entity.User;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+import pl.flyingoctopus.discord.action.DiscordAction;
+import pl.flyingoctopus.discord.action.help.DefaultHelpAction;
+import pl.flyingoctopus.discord.arguments.MessageArguments;
+import pl.flyingoctopus.discord.member.model.Member;
+import pl.flyingoctopus.discord.member.repository.MemberRepository;
+import pl.flyingoctopus.trello.service.TrelloCardService;
+import reactor.core.publisher.Mono;
+
+import java.util.regex.Pattern;
+
+@Component
+@RequiredArgsConstructor
+public class BriefAction implements DiscordAction {
+    private static final Pattern COMMAND_COMPILED_PATTERN = Pattern.compile("brief");
+
+    private final DefaultHelpAction helpAction = new DefaultHelpAction("  usage: !fo member brief <brief-report-content>");
+
+    private final MemberRepository memberRepository;
+
+    private final TrelloCardService trelloCardService;
+
+    @Override
+    public boolean isMatching(MessageArguments messageArguments) {
+        return messageArguments.getArguments().stream()
+                .findFirst()
+                .map(arg -> COMMAND_COMPILED_PATTERN.matcher(arg).matches())
+                .orElse(false);
+    }
+
+    @Override
+    public Mono<Void> run(MessageArguments messageArguments) {
+        removeFirstArgument(messageArguments);
+        return Mono.just(messageArguments.getMessage().getAuthor().get())
+                .flatMap(this::findMemberByDiscordId)
+                .filter(member -> trelloCardService.addCommenttoCard(member.getTrelloId(), member.getTrelloReportCardId(), messageArguments.getArguments().toArray(new String[0])[0]))
```

Comment:
Czemu tak wyciągasz brief? `.getArguments().toArray(new String[0])[0]))`
Arguments to lista więc powinno wystarczyć coś takiego:
```java
messageArguments.getArguments().get(0)
```
Tutaj tutorial ogólnie o generykach https://docs.oracle.com/javase/tutorial/java/generics/index.html